docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst

```diff
@@ -189,12 +189,14 @@ If a problem occurs and a task goes DEAD, view the task log files in ``$EXPTDIR/
 Generate Plots
 ^^^^^^^^^^^^^^^^^
 
-The plots are created using the graphics generation script that comes with the SRW App v2.1.0 release. Information on the plots and instructions on how to run the script can be found in :doc:`Chapter 12 <srw_v2.1.0:Graphics>` of the v2.1.0 release documentation. If the python environment is already loaded (i.e., ``(regional_workflow)`` is visible in the command prompt), users can navigate to the directory with the plotting scripts and run ``plot_allvars.py``:
+The plots are created by adding the plotting task(s) to the SRW App workflow. Instructions on how to configure these tasks can be found in :numref:`Section %s <PlotOutput>`. If the python environment is already loaded (i.e., |prompt| is visible in the command prompt), users can configure the plotting task in ``config.yaml``:
```

Comment:
The link here talks about `Plotting Configuration`, which is fine, but doesn't show the user how to run the plotting task. Since the last release, the plotting task was integrated into the workflow, so I'd suggest adding this section in with the `Configure the Verification Sample Case` section. Add `"parm/wflow/plot.yaml"` to the task group, and add the code block mentioned here (`task_plot_allvars`) to the config section.